src/utils/treasury-utils.ts

```diff
@@ -37,6 +36,15 @@ const SUPPORTED_LP_TOKENS_ADDRESSES = [
 const FRAXTAL_TOKENS = [
   '0xfc00000000000000000000000000000000000008',
   '0x8c279f6bfa31c47f29e5d05a68796f2a6c216892',
+  '0x124189d975bd59f85a0be1845b556d9249096522',
```

Comment:
Kindly update this list, we have repeated addresses

Reply:
I also think the number of added addresses should match the new treasury-data added.

Reply:
It is an exhaustive list of all tokens returned by the API, some tokens don't show up because they don't meet the minimum value threshold of 5k USD, which could change in the future, that's why I included all, wdyt?

Reply:
Oh, that's fine.